pipelines/secure-scan.yml

```diff
@@ -22,4 +35,43 @@ steps:
       EnableServices: true
       TreatSignatureUpdateFailureAs: Standard
       TreatStaleSignatureAs: Warning
-    continueOnError: true
\ No newline at end of file
+    continueOnError: true
```

Comment:
should we continueOnError here?

Reply:
Good question. Actually the PostAnalysis task below will check the logs and fail on error so I think this is correct.